src/maidsafe/vault/mpid_manager/service.h

```diff
@@ -127,14 +133,25 @@ class MpidManagerService {
   void HandleSyncedRemoveAccount(
            std::unique_ptr<MpidManager::UnresolvedRemoveAccount>&& synced_action);
 
+// =============== Account Transfer ==============================================================
+  void TransferAccount(const NodeId& destination,
+                       const std::vector<MpidManager::KVPair>& accounts);
+  void HandleAccountTransferEntry(const std::string& serialised_account,
+                                  const routing::SingleSource& sender);
+  void HandleAccountTransfer(const MpidManager::KVPair& account);
+  void HandleAccountQuery(const ImmutableData::Name& name,
+                          const NodeId& sender,
+                          const NodeId& receiver);
+
   routing::Routing& routing_;
-  mutable std::mutex accumulator_mutex_, nodes_change_mutex_, pending_account_mutex_;
+  mutable std::mutex accumulator_mutex_, nodes_change_mutex_, pending_account_mutex_, mutex_;
```

Comment:
What is this new mutex called `mutex_` protecting?  It makes it really hard to review effectively without knowing this - the other mutexes are well named for example.